platform/android/java/AndroidManifest.xml

```diff
@@ -1,58 +1,73 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-      xmlns:tools="http://schemas.android.com/tools"
-      package="com.godot.game"
-      android:versionCode="1"
-      android:versionName="1.0"
-      android:installLocation="auto"
-      >
-<supports-screens android:smallScreens="true"
-                      android:normalScreens="true"
-                      android:largeScreens="true"
-                      android:xlargeScreens="true"/>
-
-<!--glEsVersion is modified by the exporter, changing this value here has no effect-->
-    <uses-feature android:glEsVersion="0x00020000" android:required="true" />
-<!--Adding custom text to manifest is fine, but do it outside the custom user and application BEGIN/END regions, as that gets rewritten-->
-
-<!--Custom permissions XML added by add-ons. It's recommended to add them from the export preset, though-->
+    xmlns:tools="http://schemas.android.com/tools"
+    package="com.godot.game"
```

Comment:
BTW, is that `com.godot.game` package name important to keep as is? Could we maybe find a better reverse DNS name since there is no `godot.com` DNS associated with us?

It seems used for some imports in Google code that we apparently hacked a bit:
```
$ rg com.godot.game
misc/ide/jetbrains/AndroidManifest.xml
4:      package="com.godot.game"
42:                     android:targetPackage="com.godot.game" />

platform/android/java/AndroidManifest.xml
4:    package="com.godot.game"

platform/android/java/src/org/godotengine/godot/Godot.java
147:            int stringResourceID = paused ? com.godot.game.R.string.text_button_resume :
148:                                                                            com.godot.game.R.string.text_button_pause;
609:                                                    setContentView(com.godot.game.R.layout.downloading_expansion);
610:                                                    mPB = (ProgressBar)findViewById(com.godot.game.R.id.progressBar);
611:                                                    mStatusText = (TextView)findViewById(com.godot.game.R.id.statusText);
612:                                                    mProgressFraction = (TextView)findViewById(com.godot.game.R.id.progressAsFraction);
613:                                                    mProgressPercent = (TextView)findViewById(com.godot.game.R.id.progressAsPercentage);
614:                                                    mAverageSpeed = (TextView)findViewById(com.godot.game.R.id.progressAverageSpeed);
615:                                                    mTimeRemaining = (TextView)findViewById(com.godot.game.R.id.progressTimeRemaining);
616:                                                    mDashboard = findViewById(com.godot.game.R.id.downloaderDashboard);
617:                                                    mCellMessage = findViewById(com.godot.game.R.id.approveCellular);
618:                                                    mPauseButton = (Button)findViewById(com.godot.game.R.id.pauseButton);
619:                                                    mWiFiSettingsButton = (Button)findViewById(com.godot.game.R.id.wifiSettingsButton);
1089:           mAverageSpeed.setText(getString(com.godot.game.R.string.kilobytes_per_second,
1091:           mTimeRemaining.setText(getString(com.godot.game.R.string.time_remaining,

platform/android/java/src/com/google/android/vending/expansion/downloader/Helpers.java
27:import com.godot.game.R;

platform/android/java/src/com/google/android/vending/licensing/util/Base64.java
34:import com.godot.game.BuildConfig;

platform/android/java/src/com/google/android/vending/expansion/downloader/impl/DownloadNotification.java
19:import com.godot.game.R;
```

Reply:
I guess it won't be a problem if it changed all at once.